core/sandbox_helpers.go

```diff
@@ -376,7 +377,7 @@ func recoverFromCreationConflictIfNeeded(
 	client libdocker.DockerClientInterface,
 	createConfig dockertypes.ContainerCreateConfig,
 	err error,
-) (*dockercontainer.ContainerCreateCreatedBody, error) {
+) (*dockercontainer.CreateResponse, error) {
```

Comment:
Changing to these `CreateResponse` methods were the changes needed due to updating the docker/docker lib